.ci/lint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# failure is a natural part of life
+set -e
+
+if [[ "$TASK" == "rpkg" ]]; then
+    R_PACKAGE_DIR=$(pwd)/r-pkg
+    Rscript -e "install.packages('lintr')"
```

Comment:
since you've already added this install to `.ci/setup.sh`, can you remove it here?